hack/ci/cloud-init/controller.yaml.tpl

```diff
@@ -28,13 +28,13 @@
     # Pre-requisite
     ENABLED_SERVICES=key,rabbit,mysql
     # Nova
-    ENABLED_SERVICES+=,n-api,n-obj,n-cpu,n-cond,n-sch,n-novnc,n-api-meta
+    ENABLED_SERVICES+=,n-api,n-cpu,n-cond,n-sch,n-novnc,n-api-meta
     # Placement service needed for Nova
     ENABLED_SERVICES+=,placement-api,placement-client
     # Glance
-    ENABLED_SERVICES+=,g-api,g-reg
+    ENABLED_SERVICES+=,g-api
```

Comment:
I guess the Nova object store and Glance registry which were removed are gone in Wallaby? We weren't using those services anyway, so good to have them removed.

Reply:
The glance registry was deprecated in Train and removed in Wallaby, I think. Nova object store has been removed since forever